### PR TITLE
enh(a11y): Separate profile and user status user menu entries

### PR DESCRIPTION
**Pull request by Pytal**
- For https://github.com/nextcloud/server/issues/41023

## Summary

Before | After
--- | ---
image | image
image | image

## Checklist

- Code is [properly formatted](https://docs.nextcloud.com/server/latest/developer_manual/digging_deeper/continuous_integration.html#linting)
- [Sign-off message](https://github.com/src-d/guide/blob/master/developer-community/fix-DCO.md) is added to all commits
- Tests ([unit](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#unit-tests), [integration](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#integration-tests), api and/or acceptance) are included
- [x] Screenshots before/after for front-end changes
- [x] Documentation ([manuals](https://github.com/nextcloud/documentation/) or wiki) has been updated or is not required
- [x] [Backports requested](https://github.com/nextcloud/backportbot/#usage) where applicable (ex: critical bugfixes)

### CODE REVIEW

**Review by szaimen — APPROVED** [review]
Tested and works but didnt review the code

File: core/src/components/UserMenu/ProfileUserMenuEntry.vue [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+<template>
+	<li :id="id"
+		class="menu-entry">
+		<component :is="profileEnabled ? 'a' : 'span'"
+			class="menu-entry__wrapper"
+			:class="{
+				active,
+				'menu-entry__wrapper--link': profileEnabled,
+			}"
+			:href="profileEnabled ? href : null"
```
Comment (ShGKme) [inline core/src/components/UserMenu/ProfileUserMenuEntry.vue:32]: Works the same by better for type checking

```suggestion
			:href="profileEnabled ? href : undefined"
```

File: core/src/components/UserMenu/ProfileUserMenuEntry.vue [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+
+	&__content {
+		display: inline-flex !important;
+		gap: 0 10px !important;
+	}
+
+	&__displayname {
+		font-weight: bold !important;
+	}
+}
```
Comment (ShGKme) [inline core/src/components/UserMenu/ProfileUserMenuEntry.vue:139]: A number of `!important` scares here =D
Are all of them needed?

**Review by ShGKme — APPROVED** [review]
Looks fine if it is ok to move this functionality from an app to the core

**Review by JuliaKirschenheuter — APPROVED** [review]

**Pull request merged**